beads_web/src/main/java/com/beads/web/dao/OrderDaoImpl.java

```diff
@@ -0,0 +1,39 @@
+package com.beads.web.dao;
+
+import com.beads.model.domain.Order;
+import com.beads.model.domain.OrderItem;
+import com.beads.model.domain.OrderStatus;
+import java.util.List;
+import javax.persistence.criteria.CriteriaBuilder;
+import javax.persistence.criteria.CriteriaQuery;
+import javax.persistence.criteria.Fetch;
+import javax.persistence.criteria.Root;
+import org.springframework.context.annotation.Primary;
+import org.springframework.stereotype.Repository;
+
+@Repository(OrderDaoImpl.BEAN_NAME)
+@Primary
+public class OrderDaoImpl extends com.beads.model.dao.OrderDaoImpl implements OrderDao {
+
+  public static final String BEAN_NAME = "web.OrderDaoImpl";
+
+  @Override
+  public List<Order> getAllOrderByStatus(OrderStatus orderStatus) {
+    CriteriaBuilder criteriaBuilder = getSession().getCriteriaBuilder();
+    CriteriaQuery<Order> criteriaQuery = criteriaBuilder.createQuery(Order.class);
+    Root<Order> root = criteriaQuery.from(Order.class);
+    Fetch<Order, OrderItem> fetch = root.fetch("orderItems");
```

Comment:
"orderItems" looks like a constant which used in both methods. So I would suggest to extract it as a constant and use in both cases